types/base64util/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for base64util 2.2
+// Project: https://github.com/duzun/base64util
+// Definitions by: Jonathan Ehman <https://github.com/jpehman>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/* eslint-disable @typescript-eslint/naming-convention */
+export const VERSION: string;
+
+export namespace BASE64 {
+    type Prototype = unknown;
+}
+
+export function _atob(data: string): string;
+export function byteDecode(data: string): string;
+
+export function _btoa(data: string): string;
+export function byteEncode(data: string): string;
+
+export function byteUrlDecode(data: string): string;
+export function byteUrlEncode(data: string): string;
+export function decode(data: string): string;
+export function encode(data: string): string;
+export function mb2utf8(data: string): string;
+export function mbDecode(data: string): string;
+export function mbEncode(data: string): string;
+export function mbUrlDecode(data: string): string;
+export function mbUrlEncode(data: string): string;
+export function polyfill(data: string): string;
+export function urlDecode(data: string): string;
+export function urlEncode(data: string): string;
+export function utf82mb(data: string): string;
+
+export function bindProto(proto: BASE64.Prototype): void;
```

Comment:
Does this function show up in the original library? I couldn't find it when looking at the library's source (https://unpkg.com/browse/base64util@2.2.0/base64.js).

Reply:
https://github.com/duzun/base64util/blob/master/src/base64.js

It shows up here. Not sure why it wouldn't show up in unpkg.com.

Maybe even though it's being exported, it's only used internally?

Reply:
Thanks! Indeed, this function shows up on the package but for some reason not in the corresponding file in `unpkg.com`. Weird.